src/Models/CsvCatalogItem.js

```diff
@@ -239,14 +301,14 @@ CsvCatalogItem.prototype._showInCesium = function() {
             return when(featurePromise, function(results) {
                 if (defined(results)) {
```

Comment:
I skipped the adding the loop until we have the multi-feature picking since it's all pulled from in memory so no real overhead.
________________________________________
From: Kevin Ring [notifications@github.com]
Sent: Thursday, 16 April 2015 3:30 PM
To: NICTA/nationalmap
Cc: Keith Grochow
Subject: Re: [nationalmap] Region mapping updates (#635)

In src/Models/CsvCatalogItem.js<https://github.com/NICTA/nationalmap/pull/635#discussion_r28484818>:

> @@ -239,14 +301,14 @@ CsvCatalogItem.prototype._showInCesium = function() {
>              return when(featurePromise, function(results) {
>                  if (defined(results)) {


Actually, wouldn't a loop be better? Currently NM only shows the first picked feature, so updating the others is arguably unnecessary, but this is likely to change soon. See #569<https://github.com/NICTA/nationalmap/issues/569>.

�
Reply to this email directly or view it on GitHub<https://github.com/NICTA/nationalmap/pull/635/files#r28484818>.


________________________________

The information in this e-mail may be confidential and subject to legal professional privilege and/or copyright. National ICT Australia Limited accepts no liability for any damage caused by this email or its attachments.